### ISSUE TITLE
Empty Message when there are no news

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Gamer125/rssmii#3]
```
What steps will reproduce the problem?
1. Just use RSSMii

What is the expected output? What do you see instead?
If there are no news you get empty messages!


```

Original issue reported on code.google.com by `Computer...@gmail.com` on 18 Jan 2011 at 8:15

**Comment by GoogleCodeExporter** [comment]
```
The problem is that "HTTP 304 NOT MODIFIED" should be sent if there are no 
news. But the WC24-Mail-Header was is already sent when the HTTP-Header should 
get modified so that isn't possible any more. Solution: Buffer the PHP-Output
Already coded that, r29 will have a changed URL. Overnight, I'm going to that 
and, if it works, release RSSMii 2.0
```

Original comment by `Computer...@gmail.com` on 18 Jan 2011 at 8:21
- Changed state: **Started**
- Added labels: **Milestone-1.2**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with the release of RSSMii 1.2.
```

Original comment by `Computer...@gmail.com` on 19 Jan 2011 at 3:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
//ADD: Little typo in Comment 1: Of course I meant RSSMii 1.2!
```

Original comment by `Computer...@gmail.com` on 19 Jan 2011 at 4:24
